<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewAdminDashboard</class>
 <widget class="QWidget" name="ViewAdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    padding: 5px 10px;
    border-radius: 5px;
    outline: none;
    color: rgb(0, 0, 0);
    font: 700 9pt &quot;Segoe UI&quot;;
    background-color: rgb(192, 220, 161);
    border: 2px solid rgb(145, 185, 120);
}

QPushButton:hover {
    background-color: rgb(161, 184, 135); 
    border: 2px solid rgb(130, 165, 110); 
}

QPushButton:pressed {
    background-color: rgb(145, 165, 120);
    border: 2px solid rgb(130, 155, 100); 
    padding-top: 10px; 
    padding-bottom: 6px; 
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="background" native="true">
     <property name="styleSheet">
      <string notr="true">#background{
	background-color: rgb(227, 238, 219);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="bannerBackground" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#bannerBackground {
	background-image: url(:/branding/images/brand/background.jpg);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1">
         <item>
          <widget class="QLabel" name="bannerLogo">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="bannerTitle">
           <property name="styleSheet">
            <string notr="true">#bannerTitle {
	color: rgb(255, 255, 255);
	font: 700 60pt &quot;Segoe UI&quot;;
}</string>
           </property>
           <property name="text">
            <string>City Library</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>250</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="logoutButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>logout</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="adminTabs">
        <property name="styleSheet">
         <string notr="true">QTabWidget {
    border: none;
}

QTabWidget::pane {
    border: none;
	
	background-color: rgb(255, 255, 255);
}

QTabBar::tab {
    background-color: rgb(158, 206, 104);
    color: black;
    padding: 10px 20px;
    border: 1px solid rgb(138, 207, 96);
    border-radius: 5px;
    margin-left: 30px;
    margin-right: 30px;
	margin-bottom: 10px;
    font-weight: bold;
}

QTabBar::tab:hover {
    background-color: rgb(118, 193, 68);
}

QTabBar::tab:selected {
    background-color: rgb(102, 177, 43);
    border-color: rgb(74, 141, 33);
}

QTabBar::tab:!selected {
    background-color: rgb(169, 214, 141);
}

QTabWidget::indicator {
    width: 0;
    height: 0;
}
</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="accountTab">
         <property name="styleSheet">
          <string notr="true">#accountTab{
	background-color: rgb(255, 255, 255);
}</string>
         </property>
         <attribute name="title">
          <string>Manage Members</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="addMemberButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Add Member</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="memberLayout" native="true">
            <property name="styleSheet">
             <string notr="true">#memberLayout{
	background-color: rgb(255, 255, 255);
}</string>
            </property>
            <layout class="QVBoxLayout" name="memberDisplayLayout">
             <item>
              <widget class="QLabel" name="memberListLabel">
               <property name="styleSheet">
                <string notr="true">#memberListLabel{
	color: rgb(0, 0, 0);
	font: 700 12pt &quot;Segoe UI&quot;;
}</string>
               </property>
               <property name="text">
                <string>Member List</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="membersList">
               <property name="styleSheet">
                <string notr="true">#membersList{
	font: 12pt &quot;Segoe UI&quot;;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

#membersList::item {
    color: rgb(0, 0, 0);
}

#membersList::item::hover{
	border-radius: 5px;
	background-color: rgb(102, 177, 43);
	color: rgb(0, 0, 0);
}
#membersList::item::selected{
	border-radius: 5px;
	background-color: rgb(102, 177, 43);
	color: rgb(0, 0, 0);
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="catalogueTab">
         <property name="styleSheet">
          <string notr="true">#catalogueTab{
	background-color: rgb(255, 255, 255);
}</string>
         </property>
         <attribute name="title">
          <string>Manage Catalogue</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="searchLayout">
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#addButton{
	color: rgb(0, 0, 0);
	font: 700 9pt &quot;Segoe UI&quot;;
	border: 1px solid;
	background-color: rgb(192, 220, 161);
}</string>
              </property>
              <property name="text">
               <string>Add Book</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="catalogueLabel">
              <property name="styleSheet">
               <string notr="true">#catalogueLabel{
	color: rgb(0, 0, 0);
	font: 700 12pt &quot;Segoe UI&quot;;
}</string>
              </property>
              <property name="text">
               <string>Catalogue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="catalogueList">
              <property name="styleSheet">
               <string notr="true">#catalogueList{
	background-color: rgb(255, 255, 255);
}

#catalogueList::item::hover{
	background-color: rgb(102, 177, 43);
}

#catalogueList::item::selected {
	background-color: rgb(102, 177, 43);
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="footerBackground" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="footerLogo">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>472</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
